inji-web/src/__tests__/components/Common/LandingPageWrapper.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { LandingPageWrapper, LandingPageWrapperProps } from '../../../components/Common/LandingPageWrapper';
+
+// Mock the i18n configuration
+jest.mock('react-i18next', () => ({
```

Comment:
use the existing methods from mockUtils.tsx

Reply:
Used the existing method useTranslation from mockutils